tubemq-server/src/main/java/com/tencent/tubemq/server/common/utils/HasThread.java

```diff
@@ -27,6 +27,7 @@
  * The purpose of this class is to workaround Sun JVM bug #6915621, in which something internal to
  * the JDK uses Thread.currentThread() as a monitor lock. This can produce deadlocks like
  * HBASE-4367, HBASE-4101, etc.
```

Comment:
the comment should be removed or modified.

Reply:
Check it out again